### PR TITLE
Static routes

### LINKED ISSUE #349: static pages
- [ ] - `/bridge` route should be accessible without connected device 
- [ ] - 404 page should be accessible without connected device

### PR DESCRIPTION
**Pull request by slowbackspace**
close https://github.com/trezor/trezor-suite/issues/349

- bridge, onboarding, 404 page now load as static pages (without the suite loading spinner)
- bridge refactoring (now shows the page before connect is initialized)
- fixed url for downloading bridge
- moved routes definitions to suite constants, merged with staticRoutes. Now if route is static, just ass `iStatic: true` to its object.

### CODE REVIEW

File: packages/suite/src/constants/urls.ts [hunk 1]
```diff
@@ -2,3 +2,4 @@ export const TREZOR_URL = 'https://trezor.io';
 export const SUPPORT_URL = 'https://trezor.io/support/';
 export const WIKI_URL = 'https://wiki.trezor.io/';
 export const BLOG_URL = 'https://blog.trezor.io/';
+export const TREZOR_DARA_URL = 'https://wallet.trezor.io/data/';
```
Comment (mroz22) [inline packages/suite/src/constants/urls.ts]: typo!!!! :japanese_goblin:

Comment (slowbackspace) [reply packages/suite/src/constants/urls.ts]: 👌

File: packages/suite/src/utils/suite/router.ts [hunk 2]
```diff
@@ -139,5 +78,6 @@ export const isInternalRoute = (route: string) => {
 };
 
 export const isStatic = (route: string) => {
-    return routesConstants.staticRoutes.includes(route);
+    const routeFound = routes.find(r => r.pattern === route);
+    return routeFound ? !!routeFound.isStatic : true; // 404 page act as a static
```
Comment (mroz22) [inline packages/suite/src/utils/suite/router.ts:82]: why do you need !! ? dont say it doesnt work, but seems not to be needed.

Comment (slowbackspace) [reply packages/suite/src/utils/suite/router.ts:82]: beucase isStatic can be undefined, and this awesome func should return `boolean` to make sense not `boolean | undefined`.

Comment (mroz22) [reply packages/suite/src/utils/suite/router.ts:82]: ah so, makes sense now.

**Review by vladimirvolek — APPROVED** [review]

**Pull request merged**